### PR TITLE
Reorganize Sudo Section

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

This PR reorganizes the sudo section to split it from one mega page into 3 and make it more future proof. 

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: builders/manage/sudo/.pages [hunk 1]
```diff
@@ -0,0 +1,7 @@
+title: Using Sudo
+hide: false
+nav:
+  - index.md
+  - sudo.md
+  - upgrade.md
```
Comment (eshaben) [inline builders/manage/sudo/.pages:6]: these should follow the same pattern as the 'Minting Tokens': 'minting.md'

maybe 'Manage the/Your Sudo Account' and 'Dispatch Runtime Upgrades'

File: builders/manage/sudo/minting.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
+
+ - A Tanssi Appchain (Snap or Dedicated)
+ - Your Appchain's Sudo account connected to your Appchain's Polkadot.js Apps. You can refer to the [Managing Sudo guide](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank} for instructions on injecting your Sudo account into Polkadot.js Apps
+
+If you're unsure what your Tanssi Appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/minting/minting-1.webp)
```
Comment (eshaben) [inline builders/manage/sudo/minting.md:23]: the arrow looks a little bit off. it should be inline with the text its pointing to

File: builders/manage/sudo/minting.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+!!! note
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi Appchain.
+
+## Minting Tokens {: #minting-tokens }
+
+As you know, the Sudo account has the ability to perform privileged functions, including minting additional tokens. When setting up your Tanssi Appchain on [apps.tanssi.network](https://apps.tanssi.network/){target=\_blank}, you can specify genesis account balances. In other words, you have the ability to endow accounts with initial balances upon launching your Tanssi Appchain. However, you can also mint new tokens after launch with the help of the Sudo account.
```
Comment (eshaben) [inline builders/manage/sudo/minting.md:30]:
```suggestion
As you know, the Sudo account has the ability to perform privileged functions, including minting additional tokens. When setting up your Tanssi Appchain on the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}, you can specify genesis account balances. In other words, you have the ability to endow accounts with initial balances upon launching your Tanssi Appchain. However, you can also mint new tokens after launch with the help of the Sudo account.
```

this came back on a recent SEO report, we need to switch to descriptive link text as opposed to writing out the link itself

File: builders/manage/sudo/minting.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+!!! note
+    This tutorial demonstrates assigning arbitrary token balances on a TestNet Appchain that has no value. You should carefully consider the ramifications of creating additional tokens on your own Tanssi Appchain.
+
+### Checking Existing Account Balance {: #checking-existing-account-balance }
+
+The next section will demonstrate how to assign arbitrary token balances to accounts using the Sudo account. This process will overwrite the specified account's existing balance, so verifying the account is empty is a good practice before continuing. To check an account's balance, take the following steps:
+
+1. Navigate to the **Developer** tab of [Polkadot.js Apps](#configuring-polkadotjs-apps) and click on **Chain State**
```
Comment (eshaben) [inline builders/manage/sudo/minting.md:39]: There's a couple of these links, but they need to be updated to point to the page where this section is now

File: builders/manage/sudo/minting.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+### Assigning Balances with Sudo {: #assigning-balances-with-sudo }  
+
+To assign an account balance to an account, make sure to have your Sudo account accessible in [Polkadot.js Apps](#configuring-polkadotjs-apps). Then, take the following steps:
+
+1. Navigate to the **Developer** Tab of Polkadot.js Apps for your Tanssi Appchain
+2. Click on **Sudo**. If you do not see **Sudo** in this menu, then you have not associated the Sudo account with Polkadot.js Apps. Make sure that your Sudo account is injected by your wallet and connected to Polkadot.js Apps
+3. Select the **balances** pallet
+4. Select the **forceSetBalance** method
+5. Paste in the account address to endow with tokens or select it from the dropdown
+6. Enter the amount of tokens to endow the account with. In this example, we specify `9000000000000000000` for nine native tokens. Remember that Tanssi EVM Appchains have 18 decimals while Substrate or Custom Appchains decimals are configured when launching the chain. If you're unsure how many decimals your Tanssi Appchain has, navigate to the **Settings** tab and Click on **Metadata**
```
Comment (eshaben) [inline builders/manage/sudo/minting.md:57]:
```suggestion
6. Enter the amount of tokens to endow the account with. In this example, we specify `9000000000000000000` for nine native tokens. Remember that Tanssi EVM appchains have 18 decimals, while Substrate or custom appchains configure the decimals when launching the chain. If you're unsure how many decimals your Tanssi appchain has, navigate to the **Settings** tab and click on **Metadata**
```

File: builders/manage/sudo/sudo.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+For the examples in this guide, you will need to have the following:
+
+ - A Tanssi Appchain (Snap or Dedicated)
+ - Your Appchain's Sudo account connected to your Appchain's Polkadot.js Apps
+
+If you're unsure what your Tanssi Appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/sudo/sudo-1.webp)
+
+!!! note
```
Comment (eshaben) [inline builders/manage/sudo/sudo.md:27]: maybe switch this to a warning instead?

File: builders/manage/sudo/sudo.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+2. Under **in-browser account creation** select **Allow local in-browser account creation**
+3. Press **Save**
+
+![Allowing creation of in-browser storage](/images/builders/manage/sudo/sudo/sudo-2.webp)
+
+Then, head back to the accounts tab and press **Account**. You'll then be able to replace the pre-generated private key with that of your sudo account.
+
+![Adding account on Polkadot.js Apps](/images/builders/manage/sudo/sudo/sudo-3.webp)
+
+!!! note
```
Comment (eshaben) [inline builders/manage/sudo/sudo.md:44]: same here - maybe a warning

File: builders/manage/sudo/sudo.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
+---
+title: Managing Sudo
```
Comment (eshaben) [inline builders/manage/sudo/sudo.md:2]:
```suggestion
title: Managing the Sudo Account
```

File: builders/manage/sudo/sudo.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+description: Learn how to manage the sudo account of your appchain, including viewing and importing the sudo key into Polkadot.js Apps and changing the current sudo key.
+---
+
+# Using Sudo to Manage Your Appchain
```
Comment (eshaben) [inline builders/manage/sudo/sudo.md:6]:
```suggestion
# Managing Your Appchain's Sudo Account
```

File: builders/manage/sudo/upgrade.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
+---
+title: Upgrade Runtime
```
Comment (eshaben) [inline builders/manage/sudo/upgrade.md:2]:
```suggestion
title: Upgrade Your Appchain's Runtime
```

File: builders/manage/sudo/upgrade.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+description: Learn how to use the Sudo key to manage your appchain by performing the privileged action of upgrading the runtime of your Tanssi appchain.
+---
+
+# Upgrading your Appchain Runtime with Sudo
```
Comment (eshaben) [inline builders/manage/sudo/upgrade.md:6]:
```suggestion
# Upgrading Your Appchain Runtime with Sudo
```

**Review by eshaben — COMMENTED** [review]
Nice! Love the idea of splitting them out to different pages!

Augusto has a separate PR that updates all usage of "Appchain" to "appchain" (unless in a title or at the beginning of a sentence), so please make sure you're using the correct capitalization.

Please also double-check the capitalization of Sudo, I think there was at least a couple instances of using lower-case sudo.

Lastly, can you please double-check the arrows in the images to make sure they're vertically aligned?

**Comment by themacexpert** [comment]
Redirects not needed because builders/manage/sudo/ goes to index page now instead of big sudo page. (no 404)

File: builders/manage/sudo/index.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
+---
+title: Using Sudo to Manage your Appchain
```
Comment (eshaben) [inline builders/manage/sudo/index.md:2]:
```suggestion
title: Using Sudo to Manage Your Appchain
```

File: builders/manage/sudo/.pages [hunk 13]
```diff
@@ -0,0 +1,7 @@
+title: Using Sudo
+hide: false
+nav:
+  - index.md
+  - 'Manage Sudo Account': 'sudo.md'
```
Comment (eshaben) [inline builders/manage/sudo/.pages:5]:
```suggestion
  - 'Manage The Sudo Account': 'sudo.md'
```

or Your? haha

File: builders/manage/sudo/minting.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
+---
+title: Minting Native Tokens with Sudo 
+description: Learn how to use the Sudo key to manage your appchain and performing the privileged action of minting additional native tokens of your Tanssi appchain.
```
Comment (eshaben) [inline builders/manage/sudo/minting.md:3]:
```suggestion
description: Learn how to use the Sudo key to manage your appchain and perform the privileged action of minting additional native tokens of your Tanssi appchain.
```

**Review by eshaben — COMMENTED** [review]
Loooking goooood!

**Review by eshaben — COMMENTED** [review]
Awesome - the content looks great! I just noticed the Dispatch Runtime Upgrades icon looks thin in between the sudo and mint icon though. I think we're getting rid of these icons soon so won't be such a pain haha, but until then, can you please double-check the line width?

**Review by eshaben — APPROVED** [review]
Beautiful! LGTM! 🚀

**Pull request merged**